sky/packages/sky/lib/src/services/fetch.dart

```diff
@@ -53,7 +54,7 @@ Future<UrlResponse> fetchUrl(String relativeUrl) async {
   UrlRequest request = new UrlRequest()
     ..url = url
     ..autoFollowRedirects = true;
-  return await fetch(request);
+  return fetch(request);
```

Comment:
I think you want to remove the `async` keyword from the declaration to make it right.